packages/task/src/browser/task-service.ts

```diff
@@ -233,4 +233,16 @@ export class TaskService implements TaskConfigurationClient {
     protected getContext(): string | undefined {
         return this.workspaceRootUri;
     }
+
```

Comment:
@simark @akosyakov notifying you as there are changes not only in plugin-ext packages but also in `packages/task`

Reply:
ok with me